scripts/servers.py

```diff
@@ -547,7 +579,9 @@ def managed_portserver():
 
 
 @contextlib.contextmanager
-def managed_webdriver_server(chrome_version=None):
+def managed_webdriver_server(
+    chrome_version: Optional[str] = None
+) -> Iterator[psutil.Process]:
```

Comment:
I think no, because decorator `@contextlib.contextmanager` is responsible for returning ContextManager, not the method. That's why method `create_managed_web_browser` is annotated with ContextManager, not others because other methods have `contextlib.contextmanager` decorator. Please correct me If I'm missing something!

Reply:
Okay, makes sense.